include/metal/list/at.hpp

```diff
@@ -101,7 +101,7 @@ namespace metal {
                     _at<list<_70, _71, _72, _73, _74, _75, _76, _77, _78, _79>>,
                     _at<list<_80, _81, _82, _83, _84, _85, _86, _87, _88, _89>>,
                     _at<list<_90, _91, _92, _93, _94, _95, _96, _97, _98, _99>>
-                    /* clang-format on */
+                    // this comment helps clang-format
                     >;
```

Comment:
In my opinion, this is a good compromise: the comment nicely aligns the arguments of the last group. Otherwise, clang-format introduces an extra line break (because the semicolon exceeds the 80-column limit). When reading the code with only the last group misaligned I immediately start to wonder what is wrong there.

Reply:
cool trick, never occurred to me that we could trick clang-format using a comment.